assets/pages/showcase/components/breakingHeadlines.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import React from 'react';
+
+import ProductPageContentBlock from 'components/productPage/productPageContentBlock/productPageContentBlock';
+import BreakingTheHeadlines from 'components/svgs/breakingTheHeadlines';
+
+export default function BreakingHeadlines() {
+  return (
+    <ProductPageContentBlock type="grey">
+      <div className="component-product-page-text-block">
```

Comment:
Shared components take the `component` prefix in their `className`. As this is used on a specific page, it doesn't require this prefix and can take the name `product-page-text-block`.